engine/execution/ingestion/mocks/fetcher.go

```diff
@@ -0,0 +1,26 @@
+package mocks
+
+import "github.com/onflow/flow-go/model/flow"
+
+type MockFetcher struct {
+	byID map[flow.Identifier]struct{}
+}
+
+func NewMockFetcher() *MockFetcher {
+	return &MockFetcher{
+		byID: make(map[flow.Identifier]struct{}),
+	}
+}
+
+func (r *MockFetcher) FetchCollection(blockID flow.Identifier, height uint64, guarantee *flow.CollectionGuarantee) error {
+	r.byID[guarantee.ID()] = struct{}{}
+	return nil
+}
+
+func (r *MockFetcher) Force() {
```

Comment:
why this empty method?

Reply:
The real implementation will force the requests to be sent by calling `Force`. In mocks, we don't care if requests are forced to be sent, we only care the collections are actually requested. That's why I left it empty